docs/extend/extend-apiml/api-mediation-passtickets.md

```diff
@@ -3,11 +3,10 @@
 As system programmer, you can configure Zowe to use PassTickets for API services that are compatible to accept them to authenticate your service with the API Mediation Layer.
 
 ## Overview
+API clients can use either a Zowe JWT token or client certificate to access an API service even if the API service itself does not support the JWT token or client certificate.
+The authentication token is available through the API Gateway [authentication endpoint](https://docs.zowe.org/stable/extend/extend-apiml/api-mediation-security#authentication-for-api-ml-services).
```

Comment:
Nitpick: Everywhere else says `Zowe JWT token` instead of `authentication token`.

Reply:
updated to `Zowe JWT token`